app/views/catalog/_sandt_tree_show.html.haml

```diff
@@ -314,8 +314,8 @@
                       %th
                         = _("Default value")
                     %tbody
-                      - if retirement[:extra_vars]
-                        - retirement[:extra_vars].each do |key, value|
```

Comment:
@h-kataria - did this ever work?  - Or did something else change at the controller level which no longer passed `retirement` in?